### DISCUSSION TITLE
Open donation pages on other platforms with a minimum 5$/10$

### DISCUSSION THREAD

**Discussion by meotimdihia** [seaweedfs/seaweedfs#4764] [Ideas]
A minimum 50$/month is a little too much for a startup.
We have many other things too. I think @chrislusf should do it.
For example https://opencollective.com/swiper I see people donating more on OpenCollective than on Patreon.

**Comment by chrislusf** [comment]
Thanks for your suggestion! Added a $10 monthly tier.
